libpod/container_internal_linux.go

```diff
@@ -478,6 +478,10 @@ func (c *Container) checkpoint(ctx context.Context, options ContainerCheckpointO
 		return errors.Errorf("checkpointing a container requires at least CRIU %d", criu.MinCriuVersion)
 	}
 
+	if !c.runtime.ociRuntime.featureCheckCheckpointing() {
```

Comment:
Do we want to put this in restore too?

Reply:
I put the version check and the runtime feature check in one function and using that from checkpoint and restore now, so that the same checks are performed for checkpoint and restore.